tests/test_entrypoints.py

```diff
@@ -1,8 +1,18 @@
+def test_insar_gamma(script_runner):
+    ret = script_runner.run('insar_gamma', '-h')
+    assert ret.success
+
```

Comment:
Could also add a test to make sure we're passing through to the right entrypoint (maybe with a `--version` arg? `-h` won't pass through)...

This isn't blocking to me though